Docs/AcpiSamples/SSDT-EC-USBX.dsl

```diff
@@ -35,8 +35,9 @@ DefinitionBlock ("", "SSDT", 2, "ACDT", "SsdtEC", 0x00001000)
      */
 
     /**
+    
     External (_SB_.PCI0.LPCB.EC0, DeviceObj)
-
+    
```

Comment:
Why did you change whitespace here and above? Please avoid.

Reply:
ok